ethereumkit/src/main/java/org/p2p/ethereumkit/external/EthereumModule.kt

```diff
@@ -1,28 +1,36 @@
 package org.p2p.ethereumkit.external
 
 import org.koin.core.module.Module
+import org.koin.core.module.dsl.singleOf
 import org.koin.dsl.module
-import org.p2p.ethereumkit.external.balance.BalanceRepository
-import org.p2p.ethereumkit.external.balance.EthereumBalanceRepository
+import org.p2p.ethereumkit.external.balance.TokensRepository
+import org.p2p.ethereumkit.external.balance.EthereumTokensRepository
+import org.p2p.ethereumkit.external.core.CoroutineDispatchers
+import org.p2p.ethereumkit.external.core.DefaultDispatchers
 import org.p2p.ethereumkit.external.core.EthereumNetworkEnvironment
 import org.p2p.ethereumkit.external.model.EthTokenKeyProvider
+import org.p2p.ethereumkit.external.price.EthereumPriceRepository
+import org.p2p.ethereumkit.external.price.PriceRepository
 import org.p2p.ethereumkit.external.repository.EthereumKitRepository
 import org.p2p.ethereumkit.external.repository.EthereumRepository
 
-object EthereumModule {
+internal object EthereumModule {
 
     fun create(): Module = module {
-        single {
-            val seedPhrase = "apart approve black comfort steel spin real renew tone primary key cherry".split(" ")
-            EthTokenKeyProvider(seedPhrase)
-        }
-        single<EthereumRepository> { EthereumKitRepository(get(), get()) }
-        single<BalanceRepository> {
-            EthereumBalanceRepository(
+
+        single<EthereumRepository> { EthereumKitRepository(get(), get(), get()) }
+
+        single<TokensRepository> {
+            EthereumTokensRepository(
                 alchemyService = get(),
                 networkEnvironment = EthereumNetworkEnvironment.ALCHEMY,
                 gson = get()
             )
         }
+        single<PriceRepository> {
+            EthereumPriceRepository(priceApi = get())
+        }
+
+        singleOf<CoroutineDispatchers> { DefaultDispatchers() }
```

Comment:
i think this dependency would be visible in global dependency graph

Reply:
ah, i see
maybe you can extract DefaultDispatchers and CoroutineDispatchers to :core and reuse them here?

Reply:
I can, but I wanted to do not touch and moving app classes